### ISSUE TITLE
[Content organization] Content organization and/or categorization isn't clear or actionable.  (02.06.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#61940]
#### VFS team name
10-10 Health Apps

#### VFS product name
10-10EZ

#### VFS feature name
Household Information Optimization

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content organization and/or categorization isn't clear or actionable. 

### Issue Details
Several pages have helper text (sometimes styled as hint text, sometimes not -- see the other ticket from @allison0034 on that topic) placed after the `legend` but before the inputs. If a screen reader user has their screen reader announce the whole page to them, they'll hear the helper text. But more likely (especially this deep into the form), screen reader users will just be tabbing through the form inputs. Although the helper text is inside the `fieldset`, it's not announced by screen readers when tabbing to an input and may be missed entirely. 

Example: On [/deductible-expenses](https://staging.va.gov/health-care/apply/application/household-information-v2/deductible-expenses), the text "These deductible expenses will lower the amount of money we count as your income" is unlikely to be heard by screen reader users. Also true further down the page for "Funeral and burial expenses are any payments made by you, like prepaid expenses."

A user is probably still going to be able to complete the form without hearing that text. But it's possible that hearing the helper text may change how they respond, so missing that text may have some consequences for their submission.

Note that this probably isn't going to be an issue for the additional info components, because those will be a tab stop that will prompt screen reader users to open and read them (or ignore them and tab to the next thing).

### Link, screenshot or steps to recreate
Example from /deductible-expenses:
```
<fieldset class="vads-u-margin-y--2">
	<legend class="schemaform-block-title" id="root__title">Deductible expenses from 2022</legend>
	<p>These deductible expenses will lower the amount of money we count as your income.</p>
```

### VA.gov Experience Standard
[Category Number 02, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.1

### Platform Recommendation
If the helper text makes sense to be grouped with an input, then it should be placed in the `hint` property of the input component. Looking at the /deductible-expenses page, this may be an option for the funeral and burial expenses text, since it's only relevant to one field.

If the helper text is more of a general instruction to everything in the `fieldset`, the most reliable option for it to be announced across all screen readers would be to associate it with the `fieldset` using `aria-labelledby`. Something like:

```
<fieldset class="vads-u-margin-y--2" aria-labelledby="root__title unique-helper-id">
        <legend class="schemaform-block-title" id="root__title">Deductible expenses from 2022</legend>
        <p id="unique-helper-id">These deductible expenses will lower the amount of money we count as your income.</p>        
```

When a user tabs into anything in the `fieldset` for the first time, they should hear the `legend` and the helper text. See this [CodePen demo from Steve Faulkner](https://codepen.io/stevef/pen/PQMaxr).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by hdjustice** [comment]
Looking at the section as a whole, similar text is on the Spouse information page and Spouse additional information page.  We will treat all three the same for consistency.

**Comment by longmd** [comment]
@briandeconinck , this is in staging and ready for review and final sign-off. The approach was to take that description text, move it into the legend and style it differently from the actual title text.

**Comment by hdjustice** [comment]
@davidakennedy - if you have a minute today, could you take a look at this?

**Comment by davidakennedy** [comment]
@longmd @hdjustice I took a look at this and everything works as expected. I like the simplicity of adding the text to the `<legend>` vs. using `aria-` to do it.

I'm wondering if it would beneficial to add some of the paragraph text in the income sections to the legends there. Like on: https://staging.va.gov/health-care/apply/application/household-information-v2/veteran-annual-income where it defines each kind of income?

**Comment by hdjustice** [comment]
Thank you DK!

**Comment by hdjustice** [comment]
Closing ticket, will discuss additional comment separately
